index.html

```diff
@@ -49,5 +49,165 @@ <h1 class="headline-title">Hey there. I'm Rachel</h1>
+                            <ul class="works-card-skills">
+                                <li>ruby on rails</li>
+                                <li>css</li>
+                                <li>javascript</li>
+                                <li>html</li>
+                            </ul>
+                            <button type="button" class="project-btn">See Project</button>
+                        </div>
+                        </article>
+                        <article>
+                            <div class="works-card">
+                                <div class="works-card-image">
+                                    <img src="./img/Img Plaholder.svg" alt="">
+                                </div>
+                                <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
+                                <ul class="works-card-skills">
+                                    <li>ruby on rails</li>
+                                    <li>css</li>
+                                    <li>javascript</li>
+                                    <li>html</li>
+                                </ul>
+                                <button type="button" class="project-btn">See Project</button>
+                            </div>
+                            </article>
+                            <article>
+                                <div class="works-card">
+                                    <div class="works-card-image">
+                                        <img src="./img/Img Plaholder.svg" alt="">
+                                    </div>
+                                    <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
+                                    <ul class="works-card-skills">
+                                        <li>ruby on rails</li>
+                                        <li>css</li>
+                                        <li>javascript</li>
+                                        <li>html</li>
+                                    </ul>
+                                    <button type="button" class="project-btn">See Project</button>
+                                </div>
+                                </article>
+                            </div>
+                        </section>
+                        <!--ABOUT SECTION-->
+                        <section class="container-flex about-me">
+                                <div class="container">
+                                    <h2>About Me</h2>
+                                    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                                        don't hestiate to contact me.</p>
+                                    <button type="button" class="resume-btn">Get My Resume</button>
+                                </div>
```

Comment:
- [ ] As the requirements suggest, you must stick to the design as much as possible (e.g., font, colours, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- your design
 ![image](https://user-images.githubusercontent.com/52143815/192010764-804bec74-99c9-463b-85ee-8c3284a0fa5a.png)

- Figma design
![image](https://user-images.githubusercontent.com/52143815/192010916-9b9eb586-f526-4fb2-b66b-75e6c37643b3.png)